kiwi/builder/live.py

```diff
@@ -236,6 +237,15 @@ def create(self):
                 iso_header_offset, self.mbrid, self.isoname
             )
 
+        # include metadata for checkmedia tool
+        if self.checkiso is True and self.arch not in ['x86_64', 'ix86']:
+            log.warning(
+                'checkiso attribute is only supported for x86 architecture '
+                'family'
+            )
```

Comment:
I am not entirely sure about that, I was thinking on having a single kiwi description that gets build for several architectures in OBS without having to include profiles. I am not even sure if actually exists a combination (firmware, bootloader, etc.) that can be applied for isos and mulitple architectures, but that was the motivation to ignore the attribute without failing.